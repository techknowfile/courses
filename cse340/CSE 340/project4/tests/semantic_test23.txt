TYPE
  t1, t2: t;
VAR
  a, b, c: t1;
  x, y, z: t2;
{
    a = a * (b + (c - a) * (b + 1)) + 2 * c;
    z = z + x - y * (5 - x);
}

