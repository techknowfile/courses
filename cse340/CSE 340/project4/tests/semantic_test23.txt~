TYPE
  t1: t;
VAR
  a, b, c: t1;
{
    a = 1 + b;
}

