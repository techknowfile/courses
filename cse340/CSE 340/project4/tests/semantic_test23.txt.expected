BOOLEAN #
INT t1 t2 t a b c x y z #
LONG #
REAL #
STRING #
